# Autonomous Control Activation Service

bool arm
---
bool success
uint8 result